src/openstudio_lib/CMakeLists.txt

```diff
@@ -716,7 +716,7 @@ add_library(${target_name}
   ${${target_name}_qrcs}
 )
 
-set(depends
+set(${target_name}_depends
   openstudio_modeleditor
   openstudio::openstudiolib
   ${QT_LIBS}
```

Comment:
It is possible that this should be `QT_WEB_LIBS` instead:

* ResultsTabView / GeometryEditor both use QWebEngine
* 2.9.X used QT_WEB_LIBS before separation: https://github.com/NREL/OpenStudio/blob/58ed43d0b1d36eec28d167b1a82a4af9e0446c2a/openstudiocore/src/openstudio_lib/CMakeLists.txt#L741

@kyle @tijcolem thoughts?